### PR TITLE
Set up tests

### PR DESCRIPTION
**Pull request by jledentu**
This PR sets up unit tests + fixture tests.

* I copied utilities from the core `chart.js` project
* Initialize some basic unit tests
* I didn't add fixture tests, because the `fixture` util doesn't manage interactions (to do later ?)

### CODE REVIEW

**Comment by jledentu** [comment]
@benmccann Could you set up the project in Coveralls please? :)

**Comment by benmccann** [comment]
I've never set anything up in Coveralls before. I'm not sure if there's some existing Chart.js stuff that we'd like to link it to? If not, please feel free to create any accounts necessary

File: package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "url": "https://github.com/chartjs/chartjs-plugin-zoom.git"
   },
   "devDependencies": {
+    "chart.js": "^2.7.3",
```
[the remark this answers is not in the document]
Comment (jledentu) [reply package.json]: Oops, fixed!

**Comment by jledentu** [comment]
@benmccann Thanks! I created the project in Coveralls, you can see here that the code coverage is correctly generated and uploaded: https://coveralls.io/github/chartjs/chartjs-plugin-zoom

So I think that this first PR is OK to be merged.

**Comment by benmccann** [comment]
Thanks again for this! Sorry I forgot about merging it for a little while

**Pull request merged**